database-plugins/widgets/Database-batchsource.json

```diff
@@ -73,7 +73,7 @@
       "properties": [
         {
           "widget-type": "textbox",
-          "label": "Reference Name",
+          "label": "Table Name",
```

Comment:
Did you get Bhooshan's approval on this change?

Reply:
We need to revert this change